sites/cheerpx/src/content/docs/11-guides/custom-images.mdx

```diff
@@ -3,36 +3,63 @@ title: Custom disk images
 description: How to create custom ext2 images to use as a filesystem for CheerpX
 ---
 
-This guide will show you how to configure a Docker container and build it into an `ext2` image for use with CheerpX.
+This guide demonstrates how to create a custom `ext2` image for use with CheerpX while ensuring that the root ownership of files is maintained.
```

Comment:
There is no need to speak about root ownership here, it's a detail. From the perspective of the user the image must be "right", whatever that implies